code/game/objects/items/manuals.dm

```diff
@@ -199,6 +199,15 @@
 				</html>
 			"}
 
+/obj/item/book/manual/chef_recipes/double_fry

```

Comment:
This shouldn't be a child of chef_recipes or manuals in general, as these are specifically for just being books with real info. You want to use /obj/item/book/granter